apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
```

Comment:
Let's go for "The game engine for everyone."

Reply:
Copy that!